config/crd/bases/iammanager.keikoproj.io_iamroles.yaml

```diff
@@ -47,370 +49,17 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
         metadata:
-          properties:
```

Comment:
Do not commit anything from config folder until we figure out why it is removing all the content when you run make in your local machine.